sbtc/src/events.rs

```diff
@@ -350,8 +355,9 @@ impl RawTupleData {
                 // that gets emitted here.
                 vout: u32::try_from(vout).map_err(EventError::ClarityIntConversion)?,
             },
-            sweep_block_hash: BitcoinBlockHash::from_slice(&sweep_block_hash)
-                .map_err(EventError::ClarityHashConversion)?,
+            // TODO: I don't like this unwrap()
+            sweep_block_hash: BurnchainHeaderHash::from_bytes(&sweep_block_hash).unwrap(),
+            //.map_err(EventError::ClarityHashConversion)?,
```

Comment:
Let's map the error. If `EventError` doesn't have a variant for the returned error then add one.